pymess/migrations/0030_migration.py

```diff
@@ -0,0 +1,378 @@
+            name='template',
+            field=models.SlugField(blank=True, max_length=100, null=True),
+        ),
+        migrations.AlterField(
+            model_name='dialertemplatedisallowedobject',
+            name='template',
+            field=models.SlugField(blank=True, max_length=100, null=True),
+        ),
+        migrations.AlterField(
+            model_name='emailmessage',
+            name='template',
+            field=models.SlugField(blank=True, max_length=100, null=True),
+        ),
+        migrations.AlterField(
+            model_name='emailtemplateattachment',
+            name='template',
+            field=models.SlugField(blank=True, max_length=100, null=True),
+        ),
+        migrations.AlterField(
+            model_name='emailtemplatedisallowedobject',
+            name='template',
+            field=models.SlugField(blank=True, max_length=100, null=True),
+        ),
+        migrations.AlterField(
+            model_name='outputsmsmessage',
+            name='template',
+            field=models.SlugField(blank=True, max_length=100, null=True),
+        ),
+        migrations.AlterField(
+            model_name='pushnotificationmessage',
+            name='template',
+            field=models.SlugField(blank=True, max_length=100, null=True),
+        ),
+        migrations.AlterField(
+            model_name='smstemplatedisallowedobject',
+            name='template',
+            field=models.SlugField(blank=True, max_length=100, null=True),
+        ),
+
+        # 2. Rename 'template' -> 'template_old'
+
+        migrations.RenameField(
+            model_name='dialermessage',
+            old_name='template',
+            new_name='template_old',
+        ),
+        migrations.RenameField(
+            model_name='dialertemplatedisallowedobject',
+            old_name='template',
+            new_name='template_old',
+        ),
+        migrations.RenameField(
+            model_name='emailmessage',
+            old_name='template',
+            new_name='template_old',
+        ),
+        migrations.RenameField(
+            model_name='emailtemplateattachment',
+            old_name='template',
+            new_name='template_old',
+        ),
+        migrations.RenameField(
+            model_name='emailtemplatedisallowedobject',
+            old_name='template',
+            new_name='template_old',
+        ),
+        migrations.RenameField(
+            model_name='outputsmsmessage',
+            old_name='template',
+            new_name='template_old',
+        ),
+        migrations.RenameField(
+            model_name='pushnotificationmessage',
+            old_name='template',
+            new_name='template_old',
+        ),
+        migrations.RenameField(
+            model_name='smstemplatedisallowedobject',
+            old_name='template',
+            new_name='template_old',
+        ),
+
+        # We add indexes to speed-up insertion.
```

Comment:
The migration hit our 1h limit when tested. I've added indexes to see if it'd help. It didn't help.

I'm open to more suggestions on how to make this migration more efficient.